sccm/osd/deploy-use/use-a-task-sequence-to-manage-virtual-hard-disks.md

```diff
@@ -16,6 +16,10 @@ manager: dougeby
 
 *Applies to: System Center Configuration Manager (Current Branch)*
 
+
+    > [!NOTE] 
+    >  Support for this feature was deprecated in Current Branch version 1710, see [Removed and deprecated features for Configuration Manager](https://docs.microsoft.com/en-us/sccm/core/plan-design/changes/deprecated/removed-and-deprecated-cmfeatures).

```

Comment:
A few edits here:
- Remove "current branch". I generally just refer to "version yymm". 
- End that first phrase with a period. 
- For the second bit, use the standard formula: "For more information, see (link)."
- Drop the language code ("en-us/") from the URL.